frontend/pages/_app.tsx

```diff
@@ -3,13 +3,13 @@ import "bootstrap/dist/css/bootstrap.css";
 import "@fortawesome/fontawesome-svg-core/styles.css";
 import type { AppProps } from "next/app";
 import Head from "next/head";
-import React from "react";
 import PageNavbar from "../components/common/PageNavbar";
 import Footer from "../components/common/Footer";
 import DataState from "../context/DataContext/dataState";
 import { SpeechProvider } from "@speechly/react-client";
 import { MoralisProvider } from "react-moralis";
 import { NotificationProvider } from "web3uikit";
+import React from "react";
```

Comment:
details like this (implementation specification) go to the description. Why have you changed this import from line 6 to line 12 ?